models/fivetran_log__schema_changelog.sql

```diff
@@ -20,7 +50,8 @@ add_connector_info as (
         connector.destination_id,
         connector.destination_name
 
-    from schema_changes join connector using(connector_id)
+    from schema_changes join 
```

Comment:
just changed this as `using` can be volatile sometimes (mostly just left joins on snowflake but let's be consistent)